src/components/DropDownMenu.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+  export let label: string;
+  export let id: string;
+  export let options: Record<string, string>;
+  export let selected: string;
+  export let placeholder: string;
+  export let onChange = () => {};
+  export let onSelect = () => {};
+  export let required = false;
+</script>
+
+<style>
+    option[value=""][disabled] {
+      display: none;
+    }
+</style>
+
+<label class="section-title" for={id}>{label}</label>
+<div style="margin-bottom: 14px;"></div>
+<select {id} bind:value={selected} {required} on:change={onChange} on:select={onSelect} class="dropdown">
```

Comment:
does not look like the class is associated with any styles or functionality. can delete.